lighthouse-core/lib/i18n/i18n.js

```diff
@@ -9,10 +9,12 @@ const path = require('path');
 const isDeepEqual = require('lodash.isequal');
 const log = require('lighthouse-logger');
 const MessageFormat = require('intl-messageformat').default;
-const MessageParser = require('intl-messageformat-parser');
 const lookupClosestLocale = require('lookup-closest-locale');
 const LOCALES = require('./locales.js');
 
+/** @typedef {import('intl-messageformat-parser').Element} MessageElement */
```

Comment:
this is still a direct reference to what is now a transitive dependency, but the `intl-messageformat` d.ts files type these by reference into its dependencies, so I don't know any other nice way to type these but this (unless we want to do some kind of `ReturnType<MessageFormat['getAst']>['elements'][0]` :)